src/index.html

```diff
@@ -9,6 +9,64 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <link href="#home" >
```

Comment:
* It's the wrong tag usage. Read this and change the tag
https://www.w3schools.com/tags/tag_link.asp
https://www.w3schools.com/tags/tag_a.asp
* Add indentation of 2 spaces for the image tag. It will improve the readability of your code

Reply:
do I have to fork  repo from mate again?